### PR TITLE
repo sync

### PR DESCRIPTION
**Pull request by Octomerger**
This is an automated pull request to sync changes between the public and private repos.

:robot: This pull request should be merged (not squashed) to preserve continuity across repos, so please let a bot do the merging!

### CODE REVIEW

File: translations/pt-BR/content/actions/guides/building-and-testing-java-with-gradle.md [hunk 1]
```diff
@@ -38,22 +38,22 @@ Você também pode adicionar este fluxo de trabalho manualmente, criando um novo
 
 {% raw %}
 ```yaml
-name: Java CI
+nome: Java CI
 
-on: [push]
+em: [push]
```
Comment (nschonni) [inline translations/pt-BR/content/actions/guides/building-and-testing-java-with-gradle.md:43]: They appear to be translating elements in the YAML

Comment (nschonni) [reply translations/pt-BR/content/actions/guides/building-and-testing-java-with-gradle.md:43]: @chiedo maybe the revert is bringing in broken examples

**Comment by chiedo** [comment]
I'm wondering if we can close this. Repo sync has run successfully since this was created!

**Comment by nschonni** [comment]
@chiedo this was the "stuck" state I was trying to address in #1050, but it might be better to just add the autoupdate step directly to the workflow

File: translations/pt-BR/content/actions/guides/building-and-testing-java-with-gradle.md (hunk 1 above)
Comment (Pemekk) [reply translations/pt-BR/content/actions/guides/building-and-testing-java-with-gradle.md:43]: #302

**Comment by Pemekk** [comment]
C3

**Pull request merged**